src/cart/src/cart/crt_context.c

```diff
@@ -753,6 +676,15 @@ crt_req_timeout_hdlr(struct crt_rpc_priv *rpc_priv)
 		crt_rpc_complete(rpc_priv, -DER_UNREACH);
 		RPC_DECREF(rpc_priv);
 		break;
+	case RPC_STATE_QUEUED:
+		RPC_ERROR(rpc_priv,
+			  "timedout in waitq to group %s, rank %d, tgt_uri %s\n",
+			  grp_priv->gp_pub.cg_grpid,
+			  tgt_ep->ep_rank, rpc_priv->crp_tgt_uri);
```

Comment:
would help to print tgt_ep->ep_tag here as well

Reply:
ok